packages/klevu-ui/src/components/klevu-facet/klevu-facet.tsx

```diff
@@ -15,6 +15,21 @@ import {
 import { getGlobalSettings } from "../../utils/utils"
 
 export type KlevuFacetMode = "checkbox" | "radio"
+export type OPTION_TYPE = {
+  name: string;
+  value: string;
+  count: number;
+  selected: boolean;
+};
+
+const resolveFacetLabel = (option: OPTION_TYPE, facet?: KlevuFilterResultOptions) => {
+  if (facet) {
+    if(facet.type === KlevuFilterType.Rating) {
+      return <klevu-rating rating={+option.name} />
```

Comment:
Maybe use `parseInt()` instead of `+`. In case the input is something else than numbers.

```suggestion
      return <klevu-rating rating={parseInt(option.name)} />
```

Reply:
Or it could be also `parseFloat` if we allow decimals.

Reply:
changed to use parseFloat